style.css

```diff
@@ -377,3 +490,47 @@ hr {
   margin-bottom: 3px;
   margin-left: 30%;
 }
+
+.header_nav_desktop_version {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+  padding-top: 1em;
+  margin: 0;
+}
+
+.header_nav_desktop_version li {
+  list-style: none;
+  padding: 0;
+  margin: 1em;
+  font-size: 14px;
+  font-weight: bold;
+}
+
+@media only screen and (max-width: 768px) {
+  .header_nav {
+    justify-content: space-between;
+  }
+
+  .items_container {
+    width: 10%;
+  }
+
+  .header_menu {
+    display: flex;
+  }
+
+  .header_nav_desktop_version {
+    display: none;
+  }
+
+  .headline {
+    background-image: url(images/header-illsutration-mobile.png);
+  }
+
+  .enginn {
+    flex-direction: column;
+  }
```

Comment:
- [Optional] Please make sure that the web page is responsive look how it looks like when  the screen size is changed, I suggest you use more responsive sizing, like VW (try to use one responsive size in all the projects like vw for all the page) which represent the size depending on the screen width   I suggest you take a look at this [website](https://web-development.space/tools/px-to-vw/) compare the size in the design using this [website](https://web-development.space/tools/px-to-vw/)
- 
![image](https://user-images.githubusercontent.com/74744787/208267050-4cf853d7-2590-47a1-998c-2f2d121f9e3f.png)